PR1/feeds/systemCategoryName/concerts/systemFeedName/venues/feed.json

```diff
@@ -2714,9 +2714,9 @@
                         "selected": true,
                         "sensitive": false,
                         "templateProperty": null,
-                        "templateValue": "venue\\d{1,3}.csv",
+                        "templateValue": "venues\\d{1,3}.csv",
                         "userEditable": true,
-                        "value": "venues\\d{1,3}.csv"
+                        "value": "VENUES\\d{1,3}.csv"
```

Comment:
This change would catch the upper case Venues but now misses the lower case venues.. both are required!